composer.json

```diff
@@ -32,6 +44,8 @@
     },
     "scripts": {
         "test": "phpunit",
-        "test:cov": "phpunit --coverage-text --coverage-clover coverage.xml --coverage-html vendor/cov"
+        "test:cov": "phpunit --coverage-text --coverage-clover coverage.xml --coverage-html vendor/cov",
+        "cs:sniff": "tools/phpcs",
```

Comment:
is tools standard location? we may also put in `bin` folder

Reply:
Yes, it's `phive` standard installation directory.